werkzeug/test.py

```diff
@@ -240,6 +240,8 @@ class EnvironBuilder(object):
             with the key and the unpacked `tuple` items as positional
             arguments.
         -   a `str`:  The string is set as form data for the associated key.
+    -   a file-like object: The object content is loaded in memory and then
```

Comment:
Hmm, this looks the same. The `-` should be at the same column as the previous one.